src/headless-reducer-manager.ts

```diff
@@ -17,12 +17,29 @@ import createSearchStatusSlice from './slices/searchstatus';
  * Manages the current map of headless IDs to Reducers.
  */
 export default class HeadlessReducerManager {
+  /**
```

Comment:
I believe this class is internal as well

Reply:
Removed the new docs because it's internal